backend/graphql/resolvers/postingResolvers.ts

```diff
@@ -22,8 +23,9 @@ const postingResolvers = {
     posting: async (
       _parent: undefined,
       { id }: { id: string },
+      context: ExpressContext,
```

Comment:
nit:
```suggestion
      { req }: { res: Response },
```
we can only pass in what we need

Reply:
I originally tried that, but I got some type error when I did `getAccessToken(req)`, but doing `getAccessToken(context.req)` worked.